app/models/rent.rb

```diff
@@ -0,0 +1,11 @@
+class Rent < ApplicationRecord
+  validates :book, :user, :from, :to, presence: true
+  belongs_to :book, class_name: 'Book', foreign_key: 'book_id', inverse_of: :rents
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id', inverse_of: :rents
+  validate :to_cannot_be_lesser_than_from,
+           unless: proc { |rent| rent.from.nil? || rent.to.nil? }
```

Comment:
you can use `from` and `to` directly. It will reference the instance the validation is running in